docs/articles/user-guide/org-browser.md

```diff
@@ -42,3 +42,7 @@ You can retrieve a component to your local project by clicking the retrieve butt
 You can also retrieve multiple components from the default org by clicking the retrieve button next to the metadata type. If retrieving a component overwrites it, you'll be prompted to select how to proceed.
 
 ![Overwrite components](../../images/overwrite-prompt.png)
+
+When the CLI retrieves components by a metadata type, the Org Browser needs an up-to-date list of all of the components to be retrieved. Therefore, the Org Browser will automatically refresh the component list before retrieving by a type. This is so the extensions can accurately check the local workspace for existing components.
```

Comment:
@hbc-bindu let me know if you need clarification with this new doc.

cc: @lcampos

Reply:
@brpowell , reviewed the update. Let me know if the edits look ok.